<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.exceptions.InstanceMethodInvocationExceptionSpec" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-02-21T18:11:59" hostname="testing-gce-635704e8-b0eb-4898-b5e6-8d1f90074792" time="0.024">
  <properties/>
  <testcase name="Exception should have proper formatted message (1)[0]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.011"/>
  <testcase name="Exception should have proper formatted message (1)[1]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.0"/>
  <testcase name="Exception should have proper formatted message (2)[0]" classname="griffon.exceptions.InstanceMethodInvocationExceptionSpec" time="0.012"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
